apps/gitness/src/App.tsx

```diff
@@ -370,7 +370,13 @@ export default function App() {
                     },
                     {
                       path: 'rules',
-                      element: <RepoBranchSettingsRulesPageContainer />
+                      element: <RepoBranchSettingsRulesPageContainer />,
+                      children: [
+                        {
+                          path: ':identifier',
+                          element: <RepoBranchSettingsRulesPageContainer />
```

Comment:
why is this element same as the `rules` page element above? shouldn't we be rendering the "CreateRulePage" or similar?

Reply:
Not really, its the same form component...which on edit is rendered with prefilled values.